index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">

```

Comment:
Please, do not remove the meta viewport since I want the page to be fitted with other devices.

Reply:
- [ ]  Phrase the comments politely, as suggestions instead of orders.

For example:

This is a polite suggestion: "I think it would be better to use semantic tags"

This reads as an order: "Use semantic tags" or "I want the page..."